hbase-migration-tools/mirroring-client/bigtable-hbase-mirroring-client-core-parent/protobuf-java-format-shaded/pom.xml

```diff
@@ -81,6 +81,14 @@
         </configuration>
       </plugin>
 
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
```

Comment:
please add a comment that this is an internal artifact and we dont need to prevent abi breakage

Reply:
done!